tests/test_smoke.py

```diff
@@ -2646,6 +2704,66 @@ def test_invalid_names(self, invalid_name_list, store_type):
                 storage_obj = storage_lib.Storage(name=name)
                 storage_obj.add_store(store_type)
 
+    @pytest.mark.parametrize('ext_bucket_fixture, store_type',
+                             [('tmp_awscli_bucket', storage_lib.StoreType.S3),
+                              ('tmp_gsutil_bucket', storage_lib.StoreType.GCS),
+                              pytest.param('tmp_awscli_bucket_r2',
+                                           storage_lib.StoreType.R2,
+                                           marks=pytest.mark.cloudflare)])
+    def test_upload_to_existing_bucket(self, ext_bucket_fixture, request,
```

Comment:
Naming conflict - `test_upload_to_existing_bucket` already exists above

Reply:
All this portion I added should have been removed. Thanks for the catch!